aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -456,6 +443,16 @@ void TensorIterator::narrow(int dim, int64_t start, int64_t size) {
   }
 }
 
+void TensorIterator::narrow_all(int start_dim, IntList starts) {
```

Comment:
This is a bit different from [`narrow`](https://pytorch.org/docs/stable/tensors.html?highlight=narrow#torch.Tensor.narrow) in PyTorch, which takes a length. In some ways it's like [`select`](https://pytorch.org/docs/stable/tensors.html?highlight=select#torch.Tensor.select), although it leaves the dimension in the shape.